app.js

```diff
@@ -111,6 +109,11 @@ var appConfig = AppConfig(require('./package.json'));
 
 // Define availability of top accounts endpoint
 process.env.TOP = appConfig.topAccounts;
+process.env.NETWORK = appConfig.network;
+
+// Global objects to be utilized under modules/helpers where scope is not accessible
+global.constants = appConfig.constants;
```

Comment:
Yes without global its preferred. But that requires refactoring complete `helpers` namespace. As any of helper is not designed to accept `scope`. So for now we should go with `global` and when we will modularize code this change can be handled there.

Reply:
In helper, for example peer_manager, there is a singleton.
I think if you follow that it would be pretty easy to create the config manager without so much change.